packages/stream_chat_flutter/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - [#516](https://github.com/GetStream/stream-chat-flutter/issues/516): Added `StreamChatThemeData.placeholderUserImage` for 
   building a widget when the `UserAvatar` image is loading
+- Added a `backgroundColor` property to the various header widgets
```

Comment:
We should also mention various header widgets in which we did the changes.
eg: - ChannelListHeader
	- ChannelHeader

etc